### ISSUE TITLE
Installation error (glove-python-binary)

### ISSUE THREAD

**Issue by cgr71ii** [bitextor/bicleaner-ai#11]
Installation:
```bash
python -m pip install ./bicleaner-ai
```

Log:
```
Processing ./bicleaner-ai
  Preparing metadata (setup.py) ... done
Collecting scikit-learn>=0.22.1
  Using cached scikit_learn-1.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (26.5 MB)
Requirement already satisfied: PyYAML>=5.1.2 in /home/cgarcia/miniconda3/envs/glove-python/lib/python3.10/site-packages (from bicleaner-ai==1.0.1) (6.0)
Requirement already satisfied: numpy in /home/cgarcia/miniconda3/envs/glove-python/lib/python3.10/site-packages (from bicleaner-ai==1.0.1) (1.22.3)
Collecting pytest
  Using cached pytest-7.1.1-py3-none-any.whl (297 kB)
Collecting toolwrapper
  Using cached toolwrapper-2.1.0.tar.gz (3.2 kB)
  Preparing metadata (setup.py) ... done
Requirement already satisfied: joblib in /home/cgarcia/miniconda3/envs/glove-python/lib/python3.10/site-packages (from bicleaner-ai==1.0.1) (1.1.0)
Requirement already satisfied: sacremoses in /home/cgarcia/miniconda3/envs/glove-python/lib/python3.10/site-packages (from bicleaner-ai==1.0.1) (0.0.49)
Collecting bicleaner-hardrules>=2.0
  Using cached bicleaner_hardrules-2.0-py3-none-any.whl (34 kB)
Collecting sentencepiece
  Using cached sentencepiece-0.1.96-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.2 MB)
Collecting tensorflow>=2.3.2
  Using cached tensorflow-2.8.0-cp310-cp310-manylinux2010_x86_64.whl (497.6 MB)
ERROR: Could not find a version that satisfies the requirement glove-python-binary==0.2.0 (from bicleaner-ai) (from versions: none)
ERROR: No matching distribution found for glove-python-binary==0.2.0
```

The problem seems to be related to `glove-python-binary` and the python version. I have tried with python 3.7, 3.8, 3.9 and 3.10. It seems that `glove-python-binary` can't be installed with python>=3.9.

**Comment by ZJaume** [comment]
To avoid this problem, use `gensim` branch.

**Comment by cgr71ii** [comment]
Ahhh, ok, ty! Is it intended to merge this branch to master or they will keep separate?

**Comment by cgr71ii** [comment]
I have tried it, but it is not only the Rust compiler dependency what is missing. The problem is related to huggingface `tokenizers`, which now is using Rust. I have needed to run the following commands to be able to build Bicleaner AI `gensim` branch (conda build, not environment, but it should be needed as well):

```bash
conda install -c conda-forge rust # rust compiler
pip install setuptools_rust
```

The dependency `setuptools_rust` can't be directly added to the `requirements.txt` file since it needs to be installed before execute `setup.py` in order to install the Bicleaner AI dependencies (just the same topic that the installation of `setuptools`, which is automatically installed in every python/conda environment).

If I'm not wrong, these dependencies should be added to the documentation.
